### ISSUE TITLE
AGP mapping file deprecation warning

### ISSUE THREAD

**Issue by arturdryomov** [bugsnag/bugsnag-android-gradle-plugin#195]
### Expected behavior

No deprecation warning in Gradle output.

### Observed behavior

Deprecation warning in Gradle output.

### Steps to reproduce

```console
$ ./gradlew --console=plain :application:assembleRelease

...
> Task :application:shrinkReleaseRes
> Task :application:packageRelease
> Task :application:assembleRelease

> Task :application:uploadBugsnagReleaseMapping
WARNING: API 'variant.getMappingFile()' is obsolete and has been replaced with 'variant.getMappingFileProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getMappingFile(), use -Pandroid.debug.obsoleteApi=true on the command line to display more information.
Attempting upload of mapping file to Bugsnag
Bugsnag upload successful

> Task :application:bugsnagReleaseReleaseTask

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.3/userguide/command_line_interface.html#sec:command_line_warnings
...
```

BTW running with `--warning-mode all ` prints even more warnings which are unrelated to this issue.

### Version

* Gradle: 6.3
* Android Gradle plugin: 3.6.2
* Bugsnag Gradle plugin: 4.7.4

**Comment by xander-jones** [comment]
Hey @arturdryomov, thanks for reaching out to let us know about this. It's something that we're already aware of and it's on our radar to get fixed!

**Comment by ZacSweers** [comment]
It doesn't seem like there's much active development in this repo, so I've taken the initiative to open a PR for this here: #196. There are a number of other deprecated API usages though, so there are likely more warnings aside from just this one.

**Comment by mattdyoung** [comment]
Fixed in [v4.7.5](https://github.com/bugsnag/bugsnag-android-gradle-plugin/releases/tag/v4.7.5)
